src/types/fingerprint/resolvers.js

```diff
@@ -8,6 +8,10 @@ module.exports = {
       return _.descriptions[args.language];
     },
 
+    aliases: (_, args) => {
```

Comment:
This being a plural type by nature is going to make it awkward if we want to provide a way to get aliases in multiple or all languages. Hm.
(Nothing to be changed here. Just thinking out loud.)